configuring.tex

```diff
@@ -92,47 +97,53 @@ \section{Preparing for First Use: Formatting SD Cards}
 
 As an end result, we want to have a correctly formatted SD card with the essential files stored on it for MEGA65 to boot from.
 
-This is how it works: When powering on, MEGA65 will search for- and boot these files:
+This is how it works: When powering on, MEGA65 will search for, and boot up with these files:
 \begin{itemize}
-\item {\tt FREEZER.M65}
-\item {\tt AUDIOMIX.M65}
-\item {\tt C64THUMB.M65}
-\item {\tt C65THUMB.M65}
-\item {\tt MEGA65.ROM}
+\item {\tt FREEZER.M65} (freeze menu program)
+\item {\tt AUDIOMIX.M65} (freeze menu's audio mixer utility)
+\item {\tt C64THUMB.M65} (C64 thumbnail image used in freezer)
+\item {\tt C65THUMB.M65} (C65 thumbnail image used in freezer)
+\item {\tt MEGA65.ROM}   (128KB ROM file)
 \item {\tt MEGA65.D81 (default disk image, automatically mounted)}
 \end{itemize}
 
-Straight out-of-the-box, MEGA65 will only have one SD card installed, located inside behind the trap-door. This SD card contains the essential files to properly boot from.
-When an external microSD card is inserted, MEGA65 consider the external one as higher priority and will try to boot from it.
-That means that the microSD card needs to have the essential files on it, otherwise MEGA65 cannot boot properly and will fall back into loading the OpenROM which does not support all the MEGA65 features.
-In general, if MEGA65 cannot boot properly and fall back to OpenROM, it will be shown in the power-up screen, similar to this:
+Straight out-of-the-box, MEGA65 will only have one internal SD card installed, accessible via the trap-door under the case. This SD card contains all essential files needed to properly boot up.
```

Comment:
not convinced out of the box needs hyphens?

... out of the box, _the_ MEGA65

Reply:
Yep sounds good, will straighten this out.